bundles/org.openhab.ui/web/src/pages/settings/pages/home/home-edit.vue

```diff
@@ -46,41 +46,59 @@
               @updated="dirty = true" />
           </f7-col>
 
-          <f7-col v-if="modelReady && !previewMode">
-            <f7-block-title>Cards</f7-block-title>
+          <f7-col>
             <f7-segmented strong tag="p">
               <f7-button v-for="tab in modelTabs" :key="tab.value" @click="showCardControls = false; currentModelTab = tab.value" :active="currentModelTab === tab.value" :text="tab.label" />
             </f7-segmented>
 
-            <div class="display-block padding">
-              <div class="no-padding float-right">
-                <f7-button @click="showCardControls = !showCardControls" small outline :fill="showCardControls" sortable-toggle=".sortable" style="margin-top: -3px; margin-right: 5px"
-                           color="gray" icon-size="12" icon-ios="material:wrap_text" icon-md="material:wrap_text" icon-aurora="material:wrap_text">
-                  &nbsp;Reorder
-                </f7-button>
-              </div>
+            <div v-if="currentModelTab === 'locations'">
+              <config-sheet
+                :parameterGroups="locationsTabParameters.props.parameterGroups || []"
+                :parameters="locationsTabParameters.props.parameters || []"
+                :configuration="page.slots.locations[0].config"
+                @updated="dirty = true" />
+            </div>
+
+            <div v-if="currentModelTab === 'equipment'">
+              <config-sheet
+                :parameterGroups="equipmentTabParameters.props.parameterGroups || []"
+                :parameters="equipmentTabParameters.props.parameters || []"
+                :configuration="page.slots.equipment[0].config"
+                @updated="dirty = true" />
```

Comment:
I would put these config sheets below the card list actually (so there is some consistency when switching tabs from Locations & Equipment to Properties).

Reply:
I made the change but I think it could lead to a little bit less discoverability. One can think that the page ends with the dynamic list of locations, equipment...

<img width="630" alt="Screenshot 2022-04-23 at 16 59 45" src="https://user-images.githubusercontent.com/11686092/164911543-29562ac1-0c79-4378-b37a-afc0693f95ef.png">